### ISSUE TITLE
Only showing ‘per week’ option

### ISSUE THREAD

**Issue by cheniecemanning** [rosenpin/Goaly#11]
The app doesn’t allow scheduling for goals outside a week, like per month or year.

**Comment by rosenpin** [comment]
Yes, this is a feature we plan to add in a future version of the app :)  

Stay tuned for updates  
I'll edit the title to reflect the issue we plan on solving

**Comment by cubitt0** [comment]
This feature would be really helpful, are there any plans of implementing it in near future?

**Comment by rosenpin** [comment]
Monthly goals were added in Goaly 2.0. Yearly is still on the list, keeping this open.
